packages/core/src/dialects/cockroachdb/connection-manager.ts

```diff
@@ -0,0 +1,254 @@
+import pick from 'lodash/pick';
+import type { Client, ClientConfig } from 'pg';
+import type { TypeFormat, TypeId } from 'pg-types';
+import {
+  ConnectionError,
+  ConnectionRefusedError,
+  ConnectionTimedOutError,
+  HostNotFoundError,
+  HostNotReachableError,
+  InvalidConnectionError,
+} from '../../errors';
+import type { ConnectionOptions } from '../../sequelize.js';
+import { Sequelize } from '../../sequelize.js';
+import { logger } from '../../utils/logger';
+import type { Connection } from '../abstract/connection-manager';
+import { AbstractConnectionManager } from '../abstract/connection-manager';
+import type { CockroachDbDialect } from './index';
+
+export interface CockroachdbConnection extends Connection, Client {
```

Comment:
is there any difference with the postgres dialect in this file or can this just extend postgres's implementation?